meinberlin/apps/dev/management/commands/comments.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from adhocracy4.comments.models import Comment
+from meinberlin.apps.ideas.models import Idea
+
+
+class Command(BaseCommand):
+    help = """
+    Creates fake comments for testing.
+
+    The command on its own counts fake objects currently in the database and prints a summary.
+
+    Usage:
+
+        $ ./manage.py comments --user <pk> --count <int>
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--user",
+            default=0,
+            type=int,
+            help="primary key of user used to create the comments",
+        )
+        parser.add_argument(
+            "--idea",
+            default=-1,
+            type=int,
+            help="primary key of the idea to add the comments to",
```

Comment:
this argument is missing from the usage example in line 16. Also it 'd be good to add each argument in the help string.